src/planner/Query.h

```diff
@@ -627,9 +631,13 @@ class Minus final : public SetOp {
         return qctx->objPool()->add(new Minus(qctx, left, right));
     }
 
+    PlanNode* clone() const;
```

Comment:
It's better to add 'override' for this interface.

Reply:
ACK.